lib/node_modules/@stdlib/math/base/special/bessely0/test/test.js

```diff
@@ -0,0 +1,278 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var PINF = require( '@stdlib/math/constants/float64-pinf' );
+var NINF = require( '@stdlib/math/constants/float64-ninf' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var y0 = require( './../lib' );
+
+
+// FIXTURES //
+
+var hugePositive = require( './fixtures/julia/huge_positive.json' );
+var veryLargePositive = require( './fixtures/julia/very_large_positive.json' );
+var largePositive = require( './fixtures/julia/large_positive.json' );
+var mediumPositive = require( './fixtures/julia/medium_positive.json' );
+var smallPositive = require( './fixtures/julia/small_positive.json' );
+var positiveGamut = require( './fixtures/julia/positive_gamut.json' );
+var smaller = require( './fixtures/julia/smaller.json' );
+var tinyPositive = require( './fixtures/julia/tiny_positive.json' );
+var subnormal = require( './fixtures/julia/subnormal.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof y0, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function evaluates the Bessel function of the second kind of zero order (Y_0) of `x` on the interval `[50,500]`', function test( t ) {
```

Comment:
Instead of explicit intervals, use prose.

> the function evaluates the Bessel function of the second kind of zero order (Y_0) (very large positive values)

Same goes for other descriptions.